app/models/user.rb

```diff
@@ -536,6 +536,10 @@ def subscribed_to_email_follower_notifications?
     notification_setting.email_follower_notifications
   end
 
+  def reactions_to
```

Comment:
@djuber Sorry for the ping on an already merged PR. Personally, I don't find this extra method in `User` adds much (if anything) in terms of readability and `User` is big enough as is, how do you feel about removing this again?